README.md

```diff
@@ -14,16 +14,23 @@ Building for Fedora
 
 Packages needed for the build, or the build requires:
 
+* make
 * check-devel
 * cmake
 * expat-devel
 * gcc
 * libsolv-devel
 * libsolv-tools
+* librepo-devel
 * python-devel (or python3-devel for Python 3 build)
 * python-nose (or python3-nose for Python 3 build)
 * rpm-devel
 * zlib-devel
+* glib2-devel
+* gtk-doc
+* gobject-introspection-devel
+* redhat-rpm-config
```

Comment:
Why is `redhat-rpm-config` a required dependency? Is there something there that other equivalents (like `rpm-mageia-setup`) need to have?